web_m2x_options/static/src/js/form.js

```diff
@@ -394,5 +392,25 @@ odoo.define('web_m2x_options.web_m2x_options', function (require) {
                 return values;
             })
         },
+
+       render_value: function(){
+           var self = this;
+           return jQuery.when(this._super.apply(this, arguments))
+           .then(function(){
+               if(self.options.open){
+                   self.$el.find('.badge')
+                   .css('cursor', 'pointer')
+                   .click(function(e){
```

Comment:
you should call a method declared on `FieldMany2ManyTags` so it will be possible to override it.  (??? `on_badge_click` ???)